app/indexers/hyrax/indexers/file_set_indexer.rb

```diff
@@ -101,6 +101,9 @@ def to_solr # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Met
 
           # attributes set by fits for video
           solr_doc['aspect_ratio_tesim']      = file_metadata.aspect_ratio if file_metadata.aspect_ratio.present?
+
+          # support for derivatives download
+          solr_doc['extensions_and_mime_types_ssim'] = resource.extensions_and_mime_types.to_json if resource.extensions_and_mime_types.present?
```

Comment:
- The field type probably doesn't need to be indexed.
- This is doing the work of `extensions_and_mime_types` twice.
- Could/should this be stored as multiple entries in the solr field instead of a single?

Reply:
Indexing it allows us to access the information on a view without having to access the data store directly. Generally solr has been the preference for views. However, you are right that it is probably optional to do this... is it your preference to not index it?

I'm not sure I understand the benefit of storing it individually. This would generally be used when you want to access all derivatives which is how we are using it (existing methods access a derivative for a particular use).

Reply:
I meant the solr suffix should be `ssm` to avoid solr trying to analyze the contents, not that it should not be in solr at all.

Reply:
I was thinking storing as multiple values would make the solr results easier to read, but would require more processing at load time. It's not that important.